packages/fx-core/tests/question/create.test.ts

```diff
@@ -1201,7 +1201,7 @@ describe("scaffold question", () => {
             const select = question as SingleSelectQuestion;
             const options = await select.dynamicOptions!(inputs);
             assert.isTrue(options.length === 3);
-            return ok({ type: "success", result: "typescript" });
+            return ok({ type: "success", result: "python" });
```

Comment:
Is this test case for typescript? I'm not sure whether changing to python will impact tests for typescript, can you help confirm with Yuan?

Reply:
actually all languages here are the same. since ts code is not merged, we need to use python here